src/code/z_scene_proc.c

```diff
@@ -22,7 +22,7 @@ static Gfx sSceneDrawDefaultDL[] = {
 /**
  * Executes the current scene draw config handler.
  */
-void Scene_ExecuteDrawConfig(PlayState* play) {
+void Scene_Draw(PlayState* play) {
     static void (*sceneDrawConfigHandlers[])(PlayState*) = {
```

Comment:
im not too excited about this one personally. it makes it sound like its drawing more than it actually is. its more of a setup for the scene as a whole while the rooms carry the weight of drawing the actual scene/room. so I think being explicit that its drawing the config in some way is more beneficial